4.1/robot/Dockerfile

```diff
@@ -21,11 +21,14 @@ RUN pip3 install --upgrade --no-cache-dir setuptools pip virtualenv
 # version definitions
 ARG PYATS_VERSION=4.1.0
 ARG PYATS_ROBOT_VERSION=4.1.1
-ARG GENIE_VERSION=3.0.7
+ARG GENIE_VERSION=3.0.0
+ARG GENIE_PARSER_VERSION=3.0.7
 ARG GENIE_ROBOT_VERSION=3.0.8
 
```

Comment:
And this seems like it could get out of control in the future...

Reply:
yeah because installing Genie auto installs everything else. , except robot, which is separate.